src/local-storage-assignment-cache.ts

```diff
@@ -2,8 +2,14 @@ import { AssignmentCache } from '@eppo/js-client-sdk-common';
 
 import { hasWindowLocalStorage } from './configuration-factory';
 
+// noinspection JSUnusedGlobalSymbols (methods are used by common repository)
 class LocalStorageAssignmentShim {
-  LOCAL_STORAGE_KEY = 'EPPO_LOCAL_STORAGE_ASSIGNMENT_CACHE';
+  private readonly localStorageKey: string;
+
+  public constructor(storageKeySuffix?: string) {
```

Comment:
did you consider making this a required parameter?

Reply:
Thought about it. I suppose I should since we never really use it without one.

Reply:
Making this required caught a place I forgot to pass it through! Excellent suggestion!